### DISCUSSION TITLE
When the events from Postgress gets deleted?

### DISCUSSION THREAD

**Discussion by cheleswer** [svix/svix-webhooks#1108] [General]
Even after deleting the subscription, I see that events are still persisting in the Postgres DB.
Is there any certain time duration after which all these events will be deleted?  Is it configurable?

**Comment by tasn** [comment]
Events don't get deleted at the moment, only the event content. We would like to add a way to configure that though. Mind opening a ticket for this so we can have the discussion there?

**Reply by cheleswer** [reply]
1. So how does this impact the storage in postgres. Fox ex. If the system is ingesting 1k notifications to the subscribers and each notification is assumed 1kb, then how much storage it will require in postgres. 
2. If events never gets deleted automatically then I assume the storage requirement will keep growing, how to clean the storage?
3. We have integrated SVIX for webhook functionality and we can see that the "message" table data keeps growing, how to manage that?

**Reply by tasn** [reply]
First of all, let me say that we plan on adding that, please open the ticket and we will.
1. 1k * 1kb = 1mb. It would depend on your volume.
2. Correct, this is an oversight that will be remedied. It's just that given the fairly tame storage requirements and that many people want the full history, this hasn't been an issue for anyone yet (and we don't want to wipe history in prod, so it makes sense for us).
3. See the point above.

Anyhow, open a ticket, and we'll fix it.

**Reply by cheleswer** [reply]
Thank you for clarification. I have opened ticket #1111

**Reply by tasn** [reply]
Great, thanks!
